### ISSUE TITLE
Allow setting the cursor shape

### ISSUE THREAD

**Issue by lanoxx** [lanoxx/tilda#133]
This needs to be done:

 * Use `vte_terminal_set_cursor_shape` to set the cursor shape in the terminal
 * Add a drop down with these there options to the wizard:
   - VTE_CURSOR_SHAPE_BLOCK,
   - VTE_CURSOR_SHAPE_IBEAM,
   - VTE_CURSOR_SHAPE_UNDERLINE

**Comment by pik** [comment]
I think there was a patch for this by someone a while back wasn't there? I could rebase that from the current head and add wizard options.

**Comment by lanoxx** [comment]
I did not have enough time today to look into this any closer. If you could do it that it would be great.

**Comment by lanoxx** [comment]
Thank you. :)

**Comment by ghosthoughts** [comment]
Has this feature been released? I couldn't find a setting in config UI for setting cursor shape. I am using `version 1.2.4-1` in ArchLinux, last updated at `2015-06-28 19:09 UTC` as mentioned [here](https://www.archlinux.org/packages/community/x86_64/tilda/).

I am not quite concerned with UI as such. Knowing the config property name (and relevant values) is enough.
In fact, after looking into the commits referenced by this issue, I tried:

    cursor_shape = 0 (and then 1 and 2)

The process was:
1. Exit tilda
2. Add above property in `config_0`
3. Start tilda

Not only was the property not executed, but the property also was removed from the `config_0` file.

Am I missing something here?
